<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Python_IEX_GUI</class>
 <widget class="QMainWindow" name="Python_IEX_GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>739</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Python IEX GUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(197, 248, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>921</width>
      <height>651</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_user">
     <attribute name="title">
      <string>User Info</string>
     </attribute>
     <widget class="QLabel" name="connection_status">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>70</y>
        <width>451</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;
color: black;</string>
      </property>
      <property name="text">
       <string>STATUS: NOT CONNECTED</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>200</y>
        <width>281</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Enter Publishable Token</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="p_token">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>pk_23b0ae9746c642de9500a533c740e06a</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Enter Secret Token</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="s_token">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>sk_d10d4c0690a04714b60381eb680bfb5d</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="connect_button">
         <property name="styleSheet">
          <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(184, 184, 184);</string>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>21</width>
        <height>551</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QTextBrowser" name="connection_browser">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>140</y>
        <width>311</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(217, 217, 217);</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:115%; background-color:transparent;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; background-color:transparent;&quot;&gt;Initialize a connection in order to display user information&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:115%; background-color:transparent;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;All other actions that can be taken using this GUI must be done after successfully establishing a connection with the IEX REST API. &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:100%; background-color:transparent;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_daily">
     <attribute name="title">
      <string>Daily Price</string>
     </attribute>
     <widget class="QLabel" name="date_lbl">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Date</string>
      </property>
     </widget>
     <widget class="QPushButton" name="get_btn">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>93</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 5px;
border-color: rgb(12, 255, 20);
background-color: rgb(246, 254, 255);
</string>
      </property>
      <property name="text">
       <string>Get Data</string>
      </property>
     </widget>
     <widget class="QLabel" name="sym_lbl">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Symbol</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sym_in">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 5px;
border-color: rgb(12, 255, 20);
background-color: rgb(246, 254, 255);
</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="date_in">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 5px;
border-color: rgb(12, 255, 20);
background-color: rgb(246, 254, 255);
</string>
      </property>
     </widget>
     <widget class="MplGraph" name="MplGraph" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>861</width>
        <height>431</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 170);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_news">
     <attribute name="title">
      <string>News</string>
     </attribute>
     <widget class="QTextBrowser" name="news_browser">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>531</width>
        <height>551</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(201, 222, 223);</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>267</width>
        <height>137</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <property name="horizontalSpacing">
        <number>100</number>
       </property>
       <property name="verticalSpacing">
        <number>18</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Symbol</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Stories</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="news_sym">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="num_stories">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="news_button">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(176, 176, 176);</string>
         </property>
         <property name="text">
          <string>Get News</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>987</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionTrade"/>
    <addaction name="actionDie"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionTrade">
   <property name="text">
    <string>Trade</string>
   </property>
  </action>
  <action name="actionDie">
   <property name="text">
    <string>Die</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplGraph</class>
   <extends>QWidget</extends>
   <header>mplgraph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
